### ISSUE TITLE
Update to DscCommunity template (Sampler)

### ISSUE THREAD

**Issue by gaelcolas** [chocolatey-community/Chocolatey#44]
Update this Repo to the DSC Community template using sampler so that it's easier to release new versions.
Update the pipeline to run out of Azure DevOps as per usual.

**Comment by SQLDBAWithABeard** [comment]
https://dsccommunity.org/blog/convert-a-module-for-continuous-delivery/

**Comment by SQLDBAWithABeard** [comment]
We are going to need to change the default branch in the chocolatey repo to main (I think this should be a requirement anyway but thats personal) – gitversion looks for main to get the version. 

Any issues with this @gaelcolas Anyone else?

**Comment by gaelcolas** [comment]
It's a good thing to do, there's a few config changes to make but as you're updating the pipeline it's good timing.
More info here: https://dsccommunity.org/blog/convert-master-to-main/

**Comment by gaelcolas** [comment]
Yeah, Sampler now defaults to main, forgot to mention that. Let me know when you want me to change the repo if your not a Repo admin. (We should ask @pauby to make you repo admin though)

**Comment by gaelcolas** [comment]
Turns out I'm not a repo admin either... bummer...

**Comment by pauby** [comment]
@gaelcolas @SQLDBAWithABeard I've given you both Admin permissions to the repository just now to make those changes. Let me know when they are done or if there are any other problems.

**Comment by pauby** [comment]
@gaelcolas @SQLDBAWithABeard Is [this](https://github.com/chocolatey-community/Chocolatey/issues/44#issuecomment-865784997) still needed?

**Comment by SQLDBAWithABeard** [comment]
Yes @pauby  for a couple more days please, I will need to change the default branch and match to the pipeline. I will let you know when I am done
